src/main/resources/default-license-normalizer-bundle.json

```diff
@@ -57,9 +57,9 @@
     { "bundleName" : "gpl2+ce",     "licenseNamePattern" : "GNU General Public License, [v|V]ersion 2.*[c|C]lasspath [e|E]xception" },
     { "bundleName" : "gpl2+ce",     "licenseNamePattern" : "GNU General Public License, [v|V]ersion 2.*[c|C][p|P]?[e|E]" },
     { "bundleName" : "gpl2+ce",     "licenseNamePattern" : "GNU General Public License, [v|V]ersion 2.*, with the classpath exception" },
-    { "bundleName" : "gpl2+ce",     "licenseNamePattern" : "GPL2 w\/ CPE" },
-    { "bundleName" : "lgpl2.1",     "licenseUrlPattern" : ".*www.opensource.org/licenses/LGPL-2\\.1" },
-    { "bundleName" : "lgpl2.1",     "licenseUrlPattern" : ".*www.gnu.org/licenses/lgpl-2\\.1.html" },
+    { "bundleName" : "gpl2+ce",     "licenseNamePattern" : "GPL2 w/ CPE" },
+    { "bundleName" : "lgpl2.1",     "licenseUrlPattern" : ".*www\\.opensource\\.org/licenses/LGPL-2\\.1" },
+    { "bundleName" : "lgpl2.1",     "licenseUrlPattern" : ".*www\\.gnu\\.org/licenses/lgpl-2\\.1.html" },
```

Comment:
Is there still one dot unquoted at the end?

Reply:
Good catch. Fixed.